exposed-dao/src/main/kotlin/org/jetbrains/exposed/dao/References.kt

```diff
@@ -228,13 +228,15 @@ private fun <ID : Comparable<ID>> List<Entity<ID>>.preloadRelations(
     }
 }
 
-fun <SRCID : Comparable<SRCID>, SRC : Entity<SRCID>, REF : Entity<*>> Iterable<SRC>.with(vararg relations: KProperty1<out REF, Any?>): Iterable<SRC> =
+fun <SRCID : Comparable<SRCID>, SRC : Entity<SRCID>, REF : Entity<*>, L: Iterable<SRC>> L.with(vararg relations: KProperty1<out REF, Any?>): L {
```

Comment:
Would it be possible to add a comment here explaining the types, for future maintainers, please?

Reply:
Maybe in some other merge request, sorry.